### PR TITLE
Bug 2053000: [UI] incorrect layout of the download discovery iso in the dynamic plugin

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://bugzilla.redhat.com/show_bug.cgi?id=2053000

### CODE REVIEW

File: frontend/src/routes/Infrastructure/InfraEnvironments/Details/InfraEnvironmentDetailsPage.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.detail-list {
```
[the remark this answers is not in the document]
Comment (ammont82) [reply frontend/src/routes/Infrastructure/InfraEnvironments/Details/InfraEnvironmentDetailsPage.css:1]: We need to change this file? https://github.com/openshift-assisted/assisted-ui-lib/blob/master/src/common/components/ui/DetailList.tsx#L41

[the remark this answers is not in the document]
Comment (jtomasek) [reply frontend/src/routes/Infrastructure/InfraEnvironments/Details/InfraEnvironmentDetailsPage.css:1]: Yes

[the remark this answers is not in the document]
Comment (ammont82) [reply frontend/src/routes/Infrastructure/InfraEnvironments/Details/InfraEnvironmentDetailsPage.css:1]: In OCP console ACM plugins are not using App.tsx and styles are not applied. We try to find how load this styles when we use ACM plugins

**Comment by jtomasek** [comment]
Can we please close this in favour of https://github.com/stolostron/console/pull/1804

**Comment by jtomasek** [comment]
This should be closed in favour of https://github.com/stolostron/console/pull/1804

**Comment by chenz4027** [comment]
Closing as requested

**Pull request closed without merging**